src/plugins/discover/public/components/discover_grid/discover_grid_document_selection.tsx

```diff
@@ -36,6 +37,11 @@ export const SelectButton = ({ rowIndex, setCellProps }: EuiDataGridCellValueEle
   const id = useMemo(() => getDocId(doc), [doc]);
   const checked = useMemo(() => selectedDocs.includes(id), [selectedDocs, id]);
 
+  const toggleDocumentSelectionLabel = i18n.translate('discover.grid.selectDoc', {
+    defaultMessage: `Select document '{rowIndex}'`,
+    values: { rowIndex },
```

Comment:
I am wondering what would be more expected for users with screen readers: that counting starts from `0` or from `1`?

Reply:
@1Copenut any inputs on @jughosta question? I tried to look up but no definite answers. Thanks!

Reply:
For what it's worth, we add 1 / start from page `1` (not 0) in EUI for our pagination/page count announcements, so I would suggest start from 1 here as well.